### PR TITLE
feat: self-host via add-to-web3 action

### PR DESCRIPTION
**Pull request by olizilla**
see: https://github.com/web3-storage/add-to-web3/pull/1

License: (Apache-2.0 AND MIT)
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

File: .github/workflows/website.yml [hunk 1]
```diff
@@ -42,16 +42,14 @@ jobs:
...
-        uses: ipfs-shipyard/ipfs-github-action@v2.1.0
+      # Add the site to web3.storage, output the cid as `steps.ipfs.outputs.cid`
+      - name: Add to web3.storage
+        uses: web3-storage/add-to-web3@mvp
         id: ipfs
         with:
           path_to_add: packages/website/out
-          cluster_host: /dnsaddr/filecoin.storage.ipfscluster.io
-          cluster_user: ${{ secrets.CLUSTER_USER }}
-          cluster_password: ${{ secrets.CLUSTER_PASSWORD }}
```
Comment (alanshaw) [inline .github/workflows/website.yml:54]: Just a reminder to remove these two secrets from the repo after merge as this is the only thing that was using them.

**Review by alanshaw — APPROVED** [review]

**Comment by olizilla** [comment]
This is blocked on deciding how and what params to expose in ipfs-car https://github.com/web3-storage/ipfs-car/pull/61

**Comment by olizilla** [comment]
...becuase we're always wrapping with a directory right now, and the github action wants to be able to let folks pin a directory and publish it as a website, for which the wrapping directory is not wanted.

File: .github/workflows/website.yml (hunk 1 above)
Comment (olizilla) [reply .github/workflows/website.yml:54]: IT IS DONE

**Pull request merged**